apps/marketplace/pages/BriefPage.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react'
+import { withRouter, Switch, Route } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import NotFound from '../components/shared/NotFound'
+import formProps from '../components/shared/form/formPropsSelector'
+import BriefResponseForm from '../components/Brief/BriefResponseForm'
+import { loadBrief, handleBriefResponseSubmit } from '../actions/briefActions'
+
+class BriefPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      submitClicked: null
+    }
+  }
+
+  componentWillMount() {
+    const briefId = this.props.match.params.briefId
+    if (briefId.length > 0) {
+      this.props.loadInitialData(briefId)
+    }
+  }
+
+  onSubmitClicked = () => {
+    this.setState({
+      submitClicked: new Date().valueOf()
+    })
+  }
+
+  handleBriefResponseSubmit(values) {
+    const { brief } = this.props
+    this.props.handleBriefResponseSubmit(brief.id, values)
+  }
+
+  render() {
+    const { match } = this.props
+    return (
+      <div className="brief-page">
+        <Switch>
+          <Route
+            path={`${match.url}/respond`}
+            render={() =>
+              <BriefResponseForm
```

Comment:
if the initial data load fails, we don't want to load the form if you see `CreateUserPage` it shows an `ErrorBox` in this case